core/src/main/java/org/apache/iceberg/deletes/PositionDeleteIndex.java

```diff
@@ -44,4 +44,14 @@ public interface PositionDeleteIndex {
 
   /** Returns true if this collection contains no element. */
   boolean isEmpty();
+
+  /** Returns true if this collection contains elements. */
+  default boolean isNotEmpty() {
```

Comment:
I added this to avoid negation inside `if` statements. In my view, those are harder to read. Scala, for instance, has helper methods for both in collections.